stores/src/main/java/dataBase/DataBaseStorePopulator.java

```diff
@@ -5,77 +5,35 @@
 import com.issoft.training.domain.shop.categories.Book;
 import com.issoft.training.domain.shop.categories.Category;
 import com.issoft.training.domain.shop.categories.Food;
+import com.issoft.training.stores.utils.PopulatorMethod;
 import com.issoft.training.stores.utils.RandomStorePopulator;
 
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
+import java.sql.*;
 import java.util.ArrayList;
 import java.util.List;
 
-public class DataBaseStorePopulator {
+public class DataBaseStorePopulator implements PopulatorMethod {
     private Statement statement;
-    private PreparedStatement preparedStatement = null;
     private List<Category> categoryListDB;
 
-    public List<Category> createListCategoryDB() {
-        DataBaseConnect dataBaseConnect = new DataBaseConnect();
+    public List<Category> createListCategories() {
+        connection();
```

Comment:
what is the purpose of that method. whta is the meaning of that name, connection... what connection, what should i do with it...